### PR TITLE
Fixes Bug 1062441 - Add anchors

### PR DESCRIPTION
**Pull request by utkbansal**
* Adds anchors linking parts of page

### CODE REVIEW

File: bedrock/styleguide/templates/styleguide/communications/copy-rules.html [hunk 1]
```diff
@@ -28,175 +28,175 @@
...
+    <dt id="final-release">Final Release</dt>
     <dd>our internal name for the mainstream releases of Firefox and Thunderbird; do not use in user-facing communications</dd>
-    <dt>Firefox</dt>
+    <dt id="firefox">Firefox</dt>
     <dd>make sure to clearly specify whether you are talking about the Firefox family of products or one of those products (i.e. the Firefox browser) specifically; always capitalized unless appearing as part of a URL</dd>
-    <dt>Firefox Accounts</dt>
+    <dt id="firefox-accounts">Firefox Accounts</dt>
     <dd>accounts is capitalized when it follows Firefox, lowercase when on its own; it can be singular or plural as needed and is always localized</dd>
-    <dt>Firefox Add-ons</dt>
+    <dt id="firefoxadd-ons">Firefox Add-ons</dt>
```
Comment (alexgibson) [inline bedrock/styleguide/templates/styleguide/communications/copy-rules.html]: Missing a hyphen here? `firefox-add-ons`

File: bedrock/styleguide/templates/styleguide/communications/copy-rules.html [hunk 2]
```diff
@@ -28,175 +28,175 @@
...
+    <dt id="fullscreen">fullscreen</dt>
     <dd>one word, lowercase, when referring to the view mode of the browser; use “entire screen,” “whole screen” or similar formulation in other cases</dd>
-    <dt>FxOS</dt>
+    <dt id="fxos">FxOS</dt>
     <dd>for internal usage, this is the official abbreviation of “Firefox OS,” but it should not be used in any external communications</dd>
-    <dt>gear</dt>
+    <dt id="gear">gear</dt>
     <dd>used to refer to any branded merchandise, be it Mozilla, Firefox or any of our other brands; please use instead of “swag” or “merchandise”</dd>
-    <dt>http://</dt>
+    <dt id="http://">http://</dt>
```
Comment (alexgibson) [inline bedrock/styleguide/templates/styleguide/communications/copy-rules.html]: Simplify this hash to just `http`?

File: bedrock/styleguide/templates/styleguide/communications/copy-rules.html [hunk 3]
```diff
@@ -28,175 +28,175 @@
...
+    <dt id="join-mozilla">Join Mozilla</dt>
     <dd>refers specifically to the fundraising campaign and should not be used in other contexts; both words are capitalized and “Join” should never become “Joins,” “Joining” or any other form of the word</dd>
-    <dt>l10n</dt>
+    <dt id="l10n">l10n</dt>
     <dd>our abbreviation for localization (10 letters between the L and N), reserved for internal use or where space is a consideration, otherwise please avoid or explain on first use</dd>
-    <dt>left-click</dt>
+    <dt id="left-click">left-click</dt>
     <dd>hyphenated in all forms and uses</dd>
-    <dt>merch/merchandise</dt>
+    <dt id="merch/merchandise">merch/merchandise</dt>
```
Comment (alexgibson) [inline bedrock/styleguide/templates/styleguide/communications/copy-rules.html]: I think for consistency and URL readability we should generally avoid using `/` in the anchor id's. Simplify this to `merch-merchandise`?

File: bedrock/styleguide/templates/styleguide/communications/copy-rules.html [hunk 4]
```diff
@@ -28,175 +28,175 @@
...
+    <dt id="mobile-phone">mobile phone</dt>
     <dd>use only to differentiate from a tablet, otherwise use “mobile device”</dd>
-    <dt>mobile tablet</dt>
+    <dt id="mobile-tablet">mobile tablet</dt>
     <dd>redundant, do not use</dd>
-    <dt>Moz</dt>
+    <dt id="moz">Moz</dt>
     <dd>we do not generally abbreviate the Mozilla name, unless otherwise noted in this guide; please avoid</dd>
-    <dt>MozCamp/MozCamps</dt>
+    <dt id="mozcamp/mozcamps">MozCamp/MozCamps</dt>
```
Comment (alexgibson) [inline bedrock/styleguide/templates/styleguide/communications/copy-rules.html]: `mozcamp-mozcamps`

File: bedrock/styleguide/templates/styleguide/communications/copy-rules.html [hunk 5]
```diff
@@ -28,175 +28,175 @@
...
+    <dt id="mozilla">Mozilla</dt>
     <dd>make  sure Mozilla is well established early or prominently in any writing or  communications (Tabzilla satisfies this requirement, for example),  either as part of a product name (Mozilla Firefox, Mozilla Webmaker,  etc.) or on its own, but do not use any compound constructions (like  Mozilla Firefox Marketplace); even  though the Mozilla wordmark is lowercase, Mozilla should be capitalized  in all other uses, unless as part of a URL (mozilla.org)</dd>
-    <dt>Mozilla Labs</dt>
+    <dt id="mozilla-labs">Mozilla Labs</dt>
     <dd>though the Mozilla Labs wordmark is lowercase, always capitalize both words in regular text; Mozilla Labs on first mention, Labs on subsequent mentions</dd>
-    <dt>Mozilla Reps</dt>
+    <dt id="mozilla-reps">Mozilla Reps</dt>
     <dd>always two words, always capitalized</dd>
-    <dt>Mozilla Space/Spaces</dt>
+    <dt id="mozilla-space/spaces">Mozilla Space/Spaces</dt>
```
Comment (alexgibson) [inline bedrock/styleguide/templates/styleguide/communications/copy-rules.html]: Just use `mozilla-space` here?

File: bedrock/styleguide/templates/styleguide/communications/copy-rules.html [hunk 6]
```diff
@@ -28,175 +28,175 @@
...
     <dd>official terms for a specific group of Mozilla offices around the world, which include open space for the community; current locations or those in the works include San Francisco, Toronto, London, Berlin, Tokyo, Auckland, Beijing, Paris and Vancouver</dd>
-    <dt>Mozilla Webmaker</dt>
+    <dt id="mozilla-webmaker">Mozilla Webmaker</dt>
     <dd>Mozilla Webmaker on first mention, Webmaker on subsequent mentions, always capitalized</dd>
-    <dt>Mozillians</dt>
+    <dt id="mozillians">Mozillians</dt>
     <dd>refers to paid or volunteer staff of the Mozilla project;  always uppercase unless part of a URL (mozillians.org)</dd>
-    <dt>MozSpace/MozSpaces</dt>
+    <dt id="mozspace/mozspaces">MozSpace/MozSpaces</dt>
```
Comment (alexgibson) [inline bedrock/styleguide/templates/styleguide/communications/copy-rules.html]: Just use `mozspace` here?

File: bedrock/styleguide/templates/styleguide/communications/copy-rules.html (hunk 3 above)
Comment (alexgibson) [inline bedrock/styleguide/templates/styleguide/communications/copy-rules.html]: Actually, we could probably just use `merchandise` here instead to keep it simple.

File: bedrock/styleguide/templates/styleguide/communications/copy-rules.html [hunk 7]
```diff
@@ -28,175 +28,175 @@
...
     <dd>informal terms for Mozilla Spaces reserved for internal use; do not use in any communications</dd>
-    <dt>non-profit</dt>
+    <dt id="non-profit">non-profit</dt>
     <dd>though “nonprofit” is not incorrect, we prefer to use the hyphenated form</dd>
-    <dt> numbers</dt>
+    <dt id="numbers"> numbers</dt>
     <dd>spell out as words from one to nine, use numerals 10 and up; spell out large numbers like thousand, million, etc. (see also “version numbers”)</dd>
-    <dt>open/open Web</dt>
+    <dt id="open/open-web">open/open Web</dt>
```
Comment (alexgibson) [inline bedrock/styleguide/templates/styleguide/communications/copy-rules.html]: `open-web`

File: bedrock/styleguide/templates/styleguide/communications/copy-rules.html [hunk 8]
```diff
@@ -28,175 +28,175 @@
...
+    <dt id="personas">Personas</dt>
     <dd>use “Firefox Themes” instead; see that entry for usage details</dd>
-    <dt>phone</dt>
+    <dt id="phone">phone</dt>
     <dd>see “mobile phone”</dd>
-    <dt>possessives</dt>
+    <dt id="possessives">possessives</dt>
     <dd>though you may run into cases where it’s impossible, please avoid using possessives with our brands or product names where you can; see also “plurals”</dd>
-    <dt>plugin/plugins</dt>
+    <dt id="plugin/plugins">plugin/plugins</dt>
```
Comment (alexgibson) [inline bedrock/styleguide/templates/styleguide/communications/copy-rules.html]: `plugins`

File: bedrock/styleguide/templates/styleguide/communications/copy-rules.html [hunk 9]
```diff
@@ -28,175 +28,175 @@
...
+    <dt id="tabzilla">Tabzilla</dt>
     <dd>our internal name for the Mozilla universal tab that appears at the top of our sites; avoid using in user-facing communications or explain on first reference</dd>
-    <dt>themes</dt>
+    <dt id="themes">themes</dt>
     <dd>see “Firefox Themes”</dd>
-    <dt>Thunderbird</dt>
+    <dt id="thunderbird">Thunderbird</dt>
     <dd>capitalized unless appearing as part of a URL</dd>
-    <dt>Thunderbird Beta</dt>
+    <dt id="thunderbirdbeta">Thunderbird Beta</dt>
```
Comment (alexgibson) [inline bedrock/styleguide/templates/styleguide/communications/copy-rules.html]: `thunderbird-beta`

**Comment by alexgibson** [comment]
Thanks for the Pr, @utkbansal!

This looks good :+1: Just a couple of things to tidy up and I think we're good to merge this change

**Comment by utkbansal** [comment]
Thanks for the review, @alexgibson !

Should I make a new commit and then send a new pull request?

**Comment by alexgibson** [comment]
> Should I make a new commit and then send a new pull request?

No need to make a new pull request, if you just add a new commit and then push to this branch, GitHub should pick up the changes here :+1:

**Comment by utkbansal** [comment]
Done, please have a look.

**Comment by alexgibson** [comment]
Thanks for the update @utkbansal!

I squashed your two commits and merged in 2c5e49cda3f3f2e28c24667959d1291153b70a34

Thanks again for the pull request!

**Pull request closed without merging**